src/test/scala/org/ensime/intg/BasicWorkflow.scala

```diff
@@ -71,11 +71,8 @@ class BasicWorkflow extends FunSpec with Matchers {
         val javaSearchSymbol = project.rpcPublicSymbolSearch(List("java", "io", "File"), 2)
         javaSearchSymbol match {
           case SymbolSearchResults(List(
-            TypeSearchResult("java.awt.image.ImageObserver", "ImageObserver", 'class,
-              Some(_)),
             TypeSearchResult("java.io.File", "File", 'class,
-              Some(_)),
-            TypeSearchResult("org.ensime.util.FileUtilSpec", "FileUtilSpec", 'class, None))) =>
```

Comment:
Admittedly this line should never be here - the test code should not see the ensime source (I believe thats an open bug).